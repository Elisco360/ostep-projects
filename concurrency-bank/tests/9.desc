simple test: files with empty lines
