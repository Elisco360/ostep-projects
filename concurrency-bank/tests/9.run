./bank 500 tests/no_transactions_wel.txt tests/no_transactions_wel.txt
