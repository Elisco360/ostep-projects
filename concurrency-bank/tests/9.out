Opening balance: 500
Closing balance: 500
